File: PRAK403-2210817120011-MAULIDASARI.c
#include <stdio.h>
int main (){
    int i, j, a, b;

    printf("Masukkan Angka : ");
    scanf("%d %d", &a, &b);

    for(i=a, j=b; i<=b, j>=a; i++, j--){
        printf("%d %d - ", i, j);
    }
    for(i=a, j=b; i>=b, j<=a; i--, j++){
        if(a<b){
           printf("%d %d - ", i, j);
        }
        printf("", i, j);
    for(i=a, j=b; i>=b, j<=a; i--, j++){
        if(a>b){
            printf("%d %d - ", i, j);
       }
     }
   }
}

File: PRAK403-2210817120011-MAULIDASARI.py
n1=int(input('masukkan nilai 1 : '))
n2=int(input('masukkan nilai 2 : '))

i=n1
j=n2

if n1>n2:
    while i>=n2:
        print(i,j,end=' - ')
        i=i-1
        j=j+1
    else:
        while i<=n2:
            print(i,j,end=' - ')
            i=j+1
            i=j-1
